### PR TITLE
example/memcached-operator: add readme

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: example/memcached-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+metadata:
+  name: mycluster
+  namespace: default
+spec:
+  replicas: 3
+  memory: 1G
+```
+
+For this CR the operator will create memcached cluster of 2 nodes 1G memory
+each in the same namespace as the CR is defined in.
```
Comment (pipo02mix) [inline example/memcached-operator/README.md]: 2 or 3?

File: example/memcached-operator/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# memcached-operator example
+
+This is an exmpale operator using the operatorkit library.
+
+## Why memcached?
+
+[memcached][memcached] was picked as an example as this shows how the operator
+can be build to create and scale a distributed system running inside
```
Comment (rossf7) [inline example/memcached-operator/README.md]: "built" also maybe use caps for Kubernetes and Memcached.

File: example/memcached-operator/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+kubernetes. memcached is easy to scale in particular as sharding is done on the
+client side. This allows to show complete example which does not do complex
```
Comment (rossf7) [inline example/memcached-operator/README.md]: allows us to show a

File: example/memcached-operator/README.md (hunk 1 above)
Comment (kopiczko) [reply example/memcached-operator/README.md]: 3!

**Pull request merged**